taskt/UI/Forms/frmScreenRecorder.Designer.cs

```diff
@@ -60,9 +60,10 @@ private void InitializeComponent()
             this.chkGroupMovesIntoSequences.CheckState = System.Windows.Forms.CheckState.Checked;
             this.chkGroupMovesIntoSequences.Font = new System.Drawing.Font("Segoe UI", 9F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
             this.chkGroupMovesIntoSequences.ForeColor = System.Drawing.Color.AliceBlue;
-            this.chkGroupMovesIntoSequences.Location = new System.Drawing.Point(9, 223);
+            this.chkGroupMovesIntoSequences.Location = new System.Drawing.Point(12, 274);
+            this.chkGroupMovesIntoSequences.Margin = new System.Windows.Forms.Padding(4, 4, 4, 4);
             this.chkGroupMovesIntoSequences.Name = "chkGroupMovesIntoSequences";
-            this.chkGroupMovesIntoSequences.Size = new System.Drawing.Size(281, 19);
+            this.chkGroupMovesIntoSequences.Size = new System.Drawing.Size(348, 24);
```

Comment:
Please make sure that property values of the controls remain unchanged while interacting with Forms.

Reply:
I experienced the same but I somehow managed to change control names without letting it auto scale controls.